examples/declarative.groovy

```diff
@@ -1,7 +1,7 @@
 pipeline {
   agent {
     kubernetes {
-      //cloud 'kubernetes-plugin-test'
+      //cloud 'kubernetes'
```

Comment:
I guess that comment was changed by an automated IDE refactoring.
What was that comment for in the first place? Can we remove it?

Reply:
no, it's just that now the cloud is the default `kubernetes` one